src/backstop/backstop.ts

```diff
@@ -50,12 +50,14 @@ export class Backstop {
     let lp_value_promise = rpc.simulateTransaction(tx);
 
     const [config, lp_value_sim] = await Promise.all([config_promise, lp_value_promise]);
+    console.log(lp_value_sim);
     let lp_value_resp: ContractResponse<[i128, i128]> = ContractResponse.fromSimulationResponse(
       lp_value_sim,
-      tx,
+      tx.toXDR(),
       network.passphrase,
-      backstopContract.parsers['updateTokenValue']
+      backstopContract.parsers['updateTknVal']
```

Comment:
is there a reason you use the `['name']` syntax?

doesn't `parsers.updateTknVal` work?

Reply:
Yeah that works I think my copilot suggested it and I just went with it